File: src/main/java/project/entity/Customer.java
package project.entity;

public class Customer extends AbsEntity {

	private String first_name;
	private String last_name;
	private String phone;
	private String email;
	private String line_one;
	private String line_two;
	private String city;
	private String county;
	private String country;
	
	public Customer(Long id) {
	    super(id);
	}

	public Customer(Long id, String first_name, String last_name, String phone, String email, String line_one,
			String line_two, String city, String county, String country) {
		super(id);
		this.id = id;
		this.first_name = first_name;
		this.last_name = last_name;
		this.phone = phone;
		this.email = email;
		this.line_one = line_one;
		this.line_two = line_two;
		this.city = city;
		this.county = county;
		this.country = country;
	}

	public Long getId() {
		return id;
	}

	public String getFirst_name() {
		return first_name;
	}

	public String getLast_name() {
		return last_name;
	}

	public String getPhone() {
		return phone;
	}

	public String getEmail() {
		return email;
	}

	public String getLine_one() {
		return line_one;
	}

	public String getLine_two() {
		return line_two;
	}

	public String getCity() {
		return city;
	}

	public String getCounty() {
		return county;
	}

	public String getCountry() {
		return country;
	}

}

File: src/main/resources/static/js/customer.js
$(document).ready(function() {
	// dialog option
	var opt = {
		autoOpen : false,
		modal : true,
		minWidth: 320
	};

	// init three Customer form
	$('#createCustomer').dialog(opt);
	$('#createCustomer').dialog('option', 'title', 'New Customer');

	$('#editCustomer').dialog(opt);
	$('#editCustomer').dialog('option', 'title', 'Edit Customer');
	
	$('#removeCustomer').dialog(opt);
	$('#removeCustomer').dialog('option', 'title', 'Remove Customer');

	$('#addAddress').dialog(opt);
	$('#addAddress').dialog('option', 'title', 'Add Address');
	
	// create button action
	$('#btn_create').click(function() {
		$('#createCustomer').dialog('open');
	});
	
	// edit button action
	$('.rowbtn').click(function() {
		var $row = $(this).closest('tr');
		console.log($row.find('.id').val());
		
		var names = $row.find('.firstLastName').text().split(' ');
		var lines = $row.find('.lineOneTwo').text().split(' ');
		
		$('#editCustomer input[name="id"]').val($row.find('.id').val());
		$('#editCustomer input[name="firstName"]').val(names[0]);
		$('#editCustomer input[name="lastName"]').val(names[1]);
		$('#editCustomer input[name="phone"]').val($row.find('.phone').text());
		$('#editCustomer input[name="email"]').val($row.find('.email').text());
		$('#editCustomer input[name="lineOne"]').val(lines[0]);
		$('#editCustomer input[name="lineTwo"]').val(lines[1]);
		$('#editCustomer input[name="city"]').val($row.find('.city').text());
		$('#editCustomer input[name="county"]').val($row.find('.county').text());
		$('#editCustomer input[name="country"]').val($row.find('.country').text());
		
		$('#editCustomer').dialog('open');
	});
	
	// remove button action
	$('#btn_remove').click(function() {
		var ids = [];
		$('.id').each(function() {
			if ($(this).is(':checked')) {
				ids.push($(this).val());
			}
		});
		
		if (ids.length != 0 ) {
			$('#removeCustomer input[name="customerArray"]').val(ids);
			$('#removeCustomer').dialog('open');
		} else {
			infoNotice('No customer to Remove !');
		}
		
	});
	
	// add address button action
	$('#btn_addr').click(function() {
		var ids = [];
		$('.id').each(function(){
			if($(this).is(':checked')) {
				ids.push($(this).val());
			}
		});
		
		if (ids.length != 0) {
			$('#addAddress input[name="customerId"]').val(ids[0]);
			console.log($('#addAddress input[name="customerId"]').val());
			$('#addAddress').dialog('open');
		} else {
			infoNotice('No customer selected !');
		}
	});
	
	// create form handle
	$('#createCustomerForm').submit(function(e) {
		
		$('#createCustomerForm').find('input').each(function() {
			if ($.trim($(this).val()) === '' || $.trim($(this).val()) == null) {
				e.preventDefault();
			}
		});
	});
	
	var infoNotice = function(msg) {
		$('#infomsg').text(msg);
		$('#infomsg').slideDown();
		$('#infomsg').delay(500).fadeOut('fast');
	};
	
});